ek9000App/src/devEL50XX.cpp

```diff
@@ -144,21 +144,22 @@ static long el50xx_read_record(void* prec) {
 	longinRecord* precord = static_cast<longinRecord*>(prec);
 	TerminalDpvt_t* dpvt = static_cast<TerminalDpvt_t*>(precord->dpvt);
 
-	if (!dpvt || !dpvt->pterm || !dpvt->pdrv)
-		return 0;
-
-	if (!dpvt->pdrv->VerifyConnection()) {
-		recGblSetSevr(precord, COMM_ALARM, INVALID_ALARM);
-		return 0;
-	}
+	if (!util::DpvtValid(dpvt))
+		return 1;
```

Comment:
So before, each of these checked their connectivity by way of `VerifyConnection()`, but now we rely on the EK9K to track its status and let us know about a bad state by way of `getEK9000IO()` and from there we set our `COMM_ALARM`/`INVALID` severity. 

Makes sense 👍

Reply:
Yep, `VerifyConnection()` is only useful when we want to avoid calling `doModbusIO` on a disconnected device, as that results in a long timeout. The branch here was leftover from when this was using callbacks instead of buffered IO.